### PR TITLE
Add create-dmg/1.1.0

### PR DESCRIPTION
**Pull request by MartinDelille**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by toge — APPROVED** [review]

File: recipes/create-dmg/all/conanfile.py [hunk 1]
```diff
@@ -30,13 +30,15 @@ def build(self):
     def package(self):
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
         self.copy("create-dmg", dst="bin", src=self._source_subfolder)
+        self.copy("*", dst=os.path.join("bin","support"), src=os.path.join(self._source_subfolder,"support"))
```
Comment (MartinDelille) [inline recipes/create-dmg/all/conanfile.py:33]: This doesn't work yet unfortunately

Comment (SpaceIm) [inline recipes/create-dmg/all/conanfile.py:33]: Why in the bin folder. Upstream Makefile installs this folder in share folder.

**Review by SSE4 — APPROVED** [review]

File: recipes/create-dmg/all/conanfile.py (hunk 1 above)
Comment (uilianries) [reply recipes/create-dmg/all/conanfile.py:33]: @MartinDelille configuration files and other needed files, are copied to `res` folder, which is covered by Conan already.

Comment (MartinDelille) [reply recipes/create-dmg/all/conanfile.py:33]: Ok so if I understand correctly, I should replace `bin` by `res` here ?

Comment (SpaceIm) [reply recipes/create-dmg/all/conanfile.py:33]: by `res/create_dmg`

Comment (MartinDelille) [reply recipes/create-dmg/all/conanfile.py:33]: I fixed that here: https://github.com/conan-io/conan-center-index/pull/11569

**Pull request merged**